web/src/app/admin/assistants/AssistantEditor.tsx

```diff
@@ -188,8 +186,7 @@ export function AssistantEditor({
     modelOptionsByProvider.set(llmProvider.name, providerOptions);
   });
   const providerSupportingImageGenerationExists = llmProviders.some(
-    (provider) =>
-      provider.provider === "openai" || provider.provider === "anthropic"
+    (provider) => provider.provider === "openai"
```

Comment:
Can we move this somewhere more data driven? This doesn't feel like it should be hardcoded into the file.